purchase_order_variant_mgmt/wizard/purchase_manage_variant.py

```diff
@@ -91,16 +90,13 @@ def button_transfer_to_order(self):
                 order_line._onchange_quantity()
                 order_line_vals = order_line._convert_to_write(
                     order_line._cache)
-                purchase_order.order_line.create(order_line_vals)
+                purchase_order.order_line.browse().create(order_line_vals)
         lines2unlink.unlink()
 
 
 class PurchaseManageVariantLine(models.TransientModel):
     _name = 'purchase.manage.variant.line'
 
-    product_id = fields.Many2one(
-        comodel_name='product.product', string="Variant", readonly=True)
-    disabled = fields.Boolean()
```

Comment:
The product_id field was removed, so in the test file we can't do:
`  wizard.variant_line_ids.filtered(lambda x: x.product_id == product1).product_uom_qty = 0 `

Reply:
Yeah, I saw it later on your other comment. Let's continue the conversation there.